README.md

```diff
@@ -1,12 +1,11 @@
-[![Build Status](https://travis-ci.org/klarna/HiveRunner.svg?branch=release_to_maven_central)](https://travis-ci.org/klarna/HiveRunner)
 
-![ScreenShot](/images/HiveRunnerSplash.png)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.klarna/hiverunner/badge.svg?subject=com.klarna:hiverunner)](https://maven-badges.herokuapp.com/maven-central/com.klarna/hiverunner) [![Build Status](https://travis-ci.org/klarna/HiveRunner.svg?branch=master)](https://travis-ci.org/klarna/HiveRunner) ![GitHub license](https://img.shields.io/github/license/klarna/hiverunner.svg)
 
+![ScreenShot](/images/HiveRunnerSplash.png)
 
-HiveRunner
-==========
+# HiveRunner
```

Comment:
These changes are more "standard" markdown to use # for sections rather than ==== etc.